src/actions/app.js

```diff
@@ -9,14 +9,14 @@ import { createActions } from 'redux-actions';
 import { ActionTypes } from 'constants/index';
 
 export const { createSnapshot } = createActions({
-  [ActionTypes.CREATE_SNAPSHOT]: snapshot => snapshot,
-  [ActionTypes.CREATE_SNAPSHOT_JOB]: snapshot => snapshot,
-  [ActionTypes.CREATE_SNAPSHOT_SUCCESS]: snapshot => snapshot,
-  [ActionTypes.CREATE_SNAPSHOT_FAILURE]: snapshot => snapshot,
+  [ActionTypes.CREATE_SNAPSHOT]: (snapshot) => snapshot,
```

Comment:
Are these changes due to linting?

Reply:
Yes, there was a recent dependabot that updated the linter